core/templates/pages/delete-account-page/services/delete-account-backend-api.service.ts

```diff
@@ -19,10 +19,11 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { downgradeInjectable } from '@angular/upgrade/static';
-import { AppConstants } from 'app.constants';
 import { WindowRef } from 'services/contextual/window-ref.service';
 import { SiteAnalyticsService } from 'services/site-analytics.service';
 
+declare const CAN_SEND_ANALYTICS_EVENTS: boolean;
```

Comment:
I am concerned about declaring this in multiple places and confusing the global scope.

Can we keep it in constants.ts?

Reply:
So why is this line still here? It is still in global scope.